test/libsolidity/InlineAssembly.cpp

```diff
@@ -291,27 +320,27 @@ BOOST_AUTO_TEST_CASE(oversize_string_literals)
 
 BOOST_AUTO_TEST_CASE(assignment_after_tag)
 {
-	BOOST_CHECK(successParse("{ let x := 1 { tag: =: x } }"));
+	BOOST_CHECK(successParse("{ let x := 1 { 7 tag: =: x } }"));
```

Comment:
Does this mean, when you jump into `tag` you need something on at the stack top, which is going to be `x`?

If that is the case, do I need to make sure that all other local variables are just below to-be-`x`?